pyproject.toml

```diff
@@ -22,7 +22,7 @@ requires-python = ">=3.10"
 dependencies = [
     'importlib_resources; python_version < "3.11"',
     "acres",
-    "fmriprep >= 24.1.0, <= 25.0.0",
+    "fmriprep ~= 25.1.0",
```

Comment:
This is needed to relax the requirements for a transform list in `ResampleSeries`, so it can just SD unwarp a single volume.